src/store/messagesStore.js

```diff
@@ -1112,6 +1112,21 @@ const actions = {
 		// Assign the new cancel function to our data value
 		context.commit('setCancelLookForNewMessages', { cancelFunction: cancel, requestId })
 
+		// FIXME testing fetch of first messages for conversation
+		// if param is null | undefined, it won't be included in the request query
+		if (!lastKnownMessageId) {
+			console.error('Error: trying to load chat from the beginning:', {
+				token,
+				lastKnownMessageId,
+				messagesList: context.getters.messagesList(token),
+				firstKnown: context.getters.getFirstKnownMessageId(token),
+				lastKnown: context.getters.getLastKnownMessageId(token),
+				realFirstKnown: context.getters.messagesList(token)?.at(0)?.id,
+				realLastKnown: context.getters.messagesList(token)?.at(-1)?.id,
+				lastReadMessageId: context.getters.conversation(token)?.lastReadMessage,
+			})
+		}
```

Comment:
Better to log copy or serialized value. Otherwise there is a chance that when you check the log, you have a new value already by the same object reference.

Reply:
That might affect only messagesList as array; plain variables should be persistent here (and I want to check them, actually)